firecloud/fiss.py

```diff
@@ -897,7 +916,7 @@ def textify(thing):
             object_id = u'entity:%s_id' % args.entity_type
             result['__header__'] = [object_id] + list(attrs.keys())
         else:
-            result = attrs                  # Workspace attributes
+            result = attrs                  # Workspace attributes (no referenceData attributes)
```

Comment:
I think this comment modification is no longer true at this point in the code - from an earlier version of this PR?
```suggestion
            result = attrs                  # Workspace attributes
```